docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -0,0 +1,57 @@
+# Replication and Deployment in Zones
+
+## Overview
+
+M3DB supports deployment to both a single zone, with rack level isolation, and deployment across multiple zones in a region. You can also deploy M3DB across multiple regions for a global view of data, however you may experience higher ingestion latency and cross region bandwidth costs. You may want to use a replication factor of five for such a multi-region cluster to survive multiple regions suffering an outage, or be able to still upgrade hosts while a region is down.
```

Comment:
higher ingestion AND query latency right?

Also this statement is confusing "You may want to use a replication factor of five for such a multi-region cluster to survive multiple regions suffering an outage, or be able to still upgrade hosts while a region is down."

In theory that advice would apply to racks and zones as well. Why is running in 3 regions not enough? I don't think most people design their systems to tolerate 2 regions out at a time (otherwise we would run with R.F=5 in our datacenters no?)

Reply:
Yeah I'll just remove this stuff.